### DISCUSSION TITLE
Error in new instalation Using Proxmox

### DISCUSSION THREAD

**Discussion by lucianomilani** [nextcloud/all-in-one#1685] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Host OS 
Proxmox

#### Nextcloud AIO version 
Nextcloud AIO v4.0.1

![image](https://user-images.githubusercontent.com/64645578/210868926-82523898-b4b8-4d32-95f9-182c4d4147dd.png)

**Comment by szaimen** [comment]
Hi, can you post the mastercontainer logs here?

**Comment by lucianomilani** [comment]
Today i trie install 14 times...

Change LXT containers change Linux Distro trie in VM...

I put here my yml file and maistercontainer log and send for you...

OK?

Simon L. ***@***.***> escreveu no dia quinta, 5/01/2023 à(s)
20:02:

> Hi, can you post the mastercontainer logs here?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nextcloud/all-in-one/issues/1681#issuecomment-1372672974>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APNGTSRBRIEXSDAKS3IUA6LWQ4SEXANCNFSM6AAAAAATSKNUOM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 


*Luciano Milani*
Capoeirista e Pesquisador
Editor do Portal Capoeira

Telm: +351 937 777 789
Mail: ***@***.***
Site: portalcapoeira.com
Site: lucianomilani.com
*Flow:* flow.page/lucianomilani

[image: Portal Capoeira]
<http://feeds2.feedburner.com/~r/PortalCapoeira/~6/1>

**Comment by lucianomilani** [comment]
I use ARGO Tunnel  in CloudFlare

version: "3.8"

volumes:
  nextcloud_aio_mastercontainer:
    name: nextcloud_aio_mastercontainer # This line is not allowed to be
changed

services:
  nextcloud:
    image: nextcloud/all-in-one:latest # Must be changed to
'nextcloud/all-in-one:latest-arm64' when used with an arm64 CPU
    restart: always
    container_name: nextcloud-aio-mastercontainer # This line is not
allowed to be changed
    volumes:
      - nextcloud_aio_mastercontainer:/mnt/docker-aio-config # This line is
not allowed to be changed
      - /var/run/docker.sock:/var/run/docker.sock:ro # May be changed on
macOS, Windows or docker rootless. See the applicable documentation
    ports:

      - 8090:8080

    environment: # Is needed when using any of the options below
      - APACHE_PORT=11000 # Is needed when running behind a reverse proxy.
See https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md
      - APACHE_DISABLE_REWRITE_IP=1
      - NEXTCLOUD_TRUSTED_DOMAINS=cloud.milani.link 192.168.1.70
      - TRUSTED_PROXIES=192.168.1.70 # proxy host IP


i trie using this option to
      - SKIP_DOMAIN_VALIDATION=true


Simon L. ***@***.***> escreveu no dia quinta, 5/01/2023 à(s)
20:02:

> Hi, can you post the mastercontainer logs here?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nextcloud/all-in-one/issues/1681#issuecomment-1372672974>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APNGTSRBRIEXSDAKS3IUA6LWQ4SEXANCNFSM6AAAAAATSKNUOM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 


*Luciano Milani*
Capoeirista e Pesquisador
Editor do Portal Capoeira

Telm: +351 937 777 789
Mail: ***@***.***
Site: portalcapoeira.com
Site: lucianomilani.com
*Flow:* flow.page/lucianomilani

[image: Portal Capoeira]
<http://feeds2.feedburner.com/~r/PortalCapoeira/~6/1>

**Comment by szaimen** [comment]
Hi, I need the mastercontainer logs and not your docker-compose file.

you can get it with `sudo docker logs nextcloud-aio-mastercontainer`

**Comment by lucianomilani** [comment]
```
Trying to fix docker.sock permissions internally...
Creating docker group internally with id 999
Initial startup of Nextcloud All In One complete!
You should be able to open the Nextcloud AIO Interface now on port 8080 of this server!
E.g. https://internal.ip.of.this.server:8080
If your server has port 80 and 8443 open and you point a domain to your server, you can get a valid certificate automatically by opening the Nextcloud AIO Interface via:
https://your-domain-that-points-to-this-server.tld:8443
INF ts=1672949384.9840717 msg=using provided configuration config_file=/Caddyfile config_adapter=
WRN ts=1672949384.9849832 msg=Caddyfile input is not formatted; run the 'caddy fmt' command to fix inconsistencies adapter=caddyfile file=/Caddyfile line=2
INF ts=1672949384.9854307 logger=admin msg=admin endpoint started address=localhost:2019 enforce_origin=false origins=["//[::1]:2019","//127.0.0.1:2019","//localhost:2019"]
WRN ts=1672949384.9855378 logger=http msg=server is listening only on the HTTP port, so no automatic HTTPS will be applied to this server server_name=srv0 http_port=80
WRN ts=1672949384.9855444 logger=http msg=automatic HTTP->HTTPS redirects are disabled server_name=srv1
INF ts=1672949384.9856071 logger=tls.cache.maintenance msg=started background certificate maintenance cache=0xc00039c770
WRN ts=1672949384.9856281 logger=tls msg=YOUR SERVER MAY BE VULNERABLE TO ABUSE: on-demand TLS is enabled, but no protections are in place docs=https://caddyserver.com/docs/automatic-https#on-demand-tls
INF ts=1672949384.9856846 logger=http.log msg=server running name=srv0 protocols=["h1","h2","h3"]
INF ts=1672949384.9857037 logger=http msg=enabling HTTP/3 listener addr=:8443
INF ts=1672949384.9857194 logger=tls msg=cleaning storage unit description=FileStorage:/mnt/docker-aio-config/caddy/
INF ts=1672949384.985736 logger=tls msg=finished cleaning storage units
INF ts=1672949384.9857378 msg=failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 2048 kiB, got: 416 kiB). See https://github.com/lucas-clemente/quic-go/wiki/UDP-Receive-Buffer-Size for details.
INF ts=1672949384.985772 logger=http.log msg=server running name=srv1 protocols=["h1","h2","h3"]
ERR ts=1672949384.9857888 msg=unable to create folder for config autosave dir=/root/.config/caddy error=mkdir /root/.config: permission denied
INF ts=1672949384.9857938 msg=serving initial configuration
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.19.0.2. Set the 'ServerName' directive globally to suppress this message
[Thu Jan 05 20:09:44.996198 2023] [ssl:warn] [pid 132] AH01906: 172.19.0.2:8080:0 server certificate is a CA certificate (BasicConstraints: CA == TRUE !?)
[Thu Jan 05 20:09:44.996239 2023] [ssl:warn] [pid 132] AH01909: 172.19.0.2:8080:0 server certificate does NOT include an ID which matches the server name
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.19.0.2. Set the 'ServerName' directive globally to suppress this message
[Thu Jan 05 20:09:45.009397 2023] [ssl:warn] [pid 132] AH01906: 172.19.0.2:8080:0 server certificate is a CA certificate (BasicConstraints: CA == TRUE !?)
[Thu Jan 05 20:09:45.009421 2023] [ssl:warn] [pid 132] AH01909: 172.19.0.2:8080:0 server certificate does NOT include an ID which matches the server name
[Thu Jan 05 20:09:45.020976 2023] [mpm_prefork:notice] [pid 132] AH00163: Apache/2.4.54 (Debian) PHP/8.0.26 OpenSSL/1.1.1n configured -- resuming normal operations
[Thu Jan 05 20:09:45.020989 2023] [core:notice] [pid 132] AH00094: Command line: 'apache2 -D FOREGROUND'
[Thu Jan 05 20:47:30.517506 2023] [php:notice] [pid 4031] [client 127.0.0.1:59492] Slim Application Error\nType: GuzzleHttp\\Exception\\ClientException\nCode: 404\nMessage: Client error: `POST http://localhost/v1.41/containers/create?name=nextcloud-aio-nextcloud` resulted in a `404 Not Found` response:\n{&quot;message&quot;:&quot;No such image: nextcloud/aio-nextcloud:latest&quot;}\n\n\nFile: /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php\nLine: 113\nTrace: #0 /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Middleware.php(69): GuzzleHttp\\Exception\\RequestException::create(Object(GuzzleHttp\\Psr7\\Request), Object(GuzzleHttp\\Psr7\\Response), NULL, Array, NULL)\n#1 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(204): GuzzleHttp\\Middleware::GuzzleHttp\\{closure}(Object(GuzzleHttp\\Psr7\\Response))\n#2 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(153): GuzzleHttp\\Promise\\Promise::callHandler(1, Object(GuzzleHttp\\Psr7\\Response), NULL)\n#3 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/TaskQueue.php(48): GuzzleHttp\\Promise\\Promise::GuzzleHttp\\Promise\\{closure}()\n#4 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(248): GuzzleHttp\\Promise\\TaskQueue->run(true)\n#5 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(224): GuzzleHttp\\Promise\\Promise->invokeWaitFn()\n#6 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(269): GuzzleHttp\\Promise\\Promise->waitIfPending()\n#7 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(226): GuzzleHttp\\Promise\\Promise->invokeWaitList()\n#8 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(62): GuzzleHttp\\Promise\\Promise->waitIfPending()\n#9 /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Client.php(187): GuzzleHttp\\Promise\\Promise->wait()\n#10 /var/www/docker-aio/php/src/Docker/DockerActionManager.php(402): GuzzleHttp\\Client->request('POST', 'http://localhos...', Array)\n#11 /var/www/docker-aio/php/src/Controller/DockerController.php(47): AIO\\Docker\\DockerActionManager->CreateContainer(Object(AIO\\Container\\Container))\n#12 /var/www/docker-aio/php/src/Controller/DockerController.php(33): AIO\\Controller\\DockerController->PerformRecursiveContainerStart('nextcloud-aio-n...', true)\n#13 /var/www/docker-aio/php/src/Controller/DockerController.php(165): AIO\\Controller\\DockerController->PerformRecursiveContainerStart('nextcloud-aio-a...', true)\n#14 /var/www/docker-aio/php/src/Controller/DockerController.php(149): AIO\\Controller\\DockerController->startTopContainer(true)\n#15 /var/www/docker-aio/php/vendor/slim/slim/Slim/Handlers/Strategies/RequestResponse.php(38): AIO\\Controller\\DockerController->StartContainer(Object(GuzzleHttp\\Psr7\\ServerRequest), Object(GuzzleHttp\\Psr7\\Response), Array)\n#16 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(358): Slim\\Handlers\\Strategies\\RequestResponse->__invoke(Array, Object(GuzzleHttp\\Psr7\\ServerRequest), Object(GuzzleHttp\\Psr7\\Response), Array)\n#17 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\\Routing\\Route->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#18 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\\MiddlewareDispatcher->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#19 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(315): Slim\\MiddlewareDispatcher->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#20 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/RouteRunner.php(68): Slim\\Routing\\Route->run(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#21 /var/www/docker-aio/php/vendor/slim/csrf/src/Guard.php(476): Slim\\Routing\\RouteRunner->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#22 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(168): Slim\\Csrf\\Guard->process(Object(GuzzleHttp\\Psr7\\ServerRequest), Object(Slim\\Routing\\RouteRunner))\n#23 /var/www/docker-aio/php/vendor/slim/twig-view/src/TwigMiddleware.php(115): Psr\\Http\\Server\\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#24 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\\Views\\TwigMiddleware->process(Object(GuzzleHttp\\Psr7\\ServerRequest), Object(Psr\\Http\\Server\\RequestHandlerInterface@anonymous))\n#25 /var/www/docker-aio/php/src/Middleware/AuthMiddleware.php(38): Psr\\Http\\Server\\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#26 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(269): AIO\\Middleware\\AuthMiddleware->__invoke(Object(GuzzleHttp\\Psr7\\ServerRequest), Object(Psr\\Http\\Server\\RequestHandlerInterface@anonymous))\n#27 /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(76): Psr\\Http\\Server\\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#28 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\\Middleware\\ErrorMiddleware->process(Object(GuzzleHttp\\Psr7\\ServerRequest), Object(Psr\\Http\\Server\\RequestHandlerInterface@anonymous))\n#29 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Psr\\Http\\Server\\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#30 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(199): Slim\\MiddlewareDispatcher->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#31 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(183): Slim\\App->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#32 /var/www/docker-aio/php/public/index.php(167): Slim\\App->run()\n#33 {main}, referer: https://192.168.1.71:8090/containers
Deleting duplicate sessions
```

**Comment by szaimen** [comment]
Interesting! Can you try to run `sudo docker exec -it nextcloud-aio-mastercontainer docker pull nextcloud/aio-nextcloud:latest` and post the output here?

Also what is the output of `sudo docker inspect nextcloud/aio-nextcloud:latest`?

**Comment by lucianomilani** [comment]
root@nextcloud:~/docker/nextcloud# sudo docker exec -it nextcloud-aio-mastercontainer docker pull nextcloud/aio-nextcloud:latest
latest: Pulling from nextcloud/aio-nextcloud
ca7dd9ec2225: Already exists 
1b78b4fe0ca1: Pull complete 
9d6040f2a28f: Pull complete 
0e2e66b89284: Pull complete 
05497c4b6f53: Pull complete 
09a30d168d1f: Pull complete 
8fb384cc931b: Pull complete 
a8b7a0d9ae62: Pull complete 
74527b27d94c: Pull complete 
b8f1bf3aa8e6: Pull complete 
31a5c5a4b65c: Pull complete 
4b627c8c229b: Pull complete 
7e4c9d93ecfa: Pull complete 
c4c8eae7becc: Pull complete 
aef16262d39d: Pull complete 
ad53494b4bf0: Pull complete 
2dec2f6f5f2c: Pull complete 
7ccf33fad8f6: Pull complete 
1b9c651f3bf6: Pull complete 
3fb51e5f9373: Pull complete 
74a9382fc3a2: Pull complete 
3080c8bf0552: Pull complete 
4f5704c75816: Pull complete 
63b4a0a569ce: Pull complete 
b07b06537e53: Extracting [==================================================>]  176.5MB/176.5MB
4f4fb700ef54: Download complete 
37d30a3206cf: Download complete 
8a02ac8382d2: Download complete 
7d6dbdd25b7e: Download complete 
failed to register layer: ApplyLayer exit status 1 stdout:  stderr: unlinkat /usr/src/nextcloud/apps/updatenotification/appinfo: invalid argument
root@nextcloud:~/docker/nextcloud#

**Comment by szaimen** [comment]
```
Download complete failed to register layer: ApplyLayer exit status 1 
stdout: stderr: unlinkat /usr/src/nextcloud/apps/updatenotification/appinfo: invalid argument 
```

We have found your issue...

**Comment by lucianomilani** [comment]
root@nextcloud:~# sudo docker inspect nextcloud/aio-nextcloud:latest
[]
Error: No such object: nextcloud/aio-nextcloud:latest

**Comment by lucianomilani** [comment]
> ```
> Download complete failed to register layer: ApplyLayer exit status 1 
> stdout: stderr: unlinkat /usr/src/nextcloud/apps/updatenotification/appinfo: invalid argument 
> ```
> 
> We have found your issue...

And now???

**Comment by szaimen** [comment, marked as answer]
https://www.google.com/search?q=Download%20complete%20failed%20to%20register%20layer%3A%20ApplyLayer%20exit%20status%201

**Reply by lucianomilani** [reply]
LXT Disk alocate is 900GB
I change Disk to local Proxmox disk and execute again the command... is OK
I trie again install NCAIO

**Comment by ricardopinto** [comment]
@lucianomilani I followed these instructions and it worked for me. you might want to try: https://forum.proxmox.com/threads/docker-failed-to-register-layer-applylayer-exit-status-1-stdout-stderr-unlinkat-var-log-apt-invalid-argument.119954/post-521157
